srcflib/plumbing/bespoke.py

```diff
@@ -32,7 +32,7 @@
 
 
 @contextmanager
-def context(sess: SQLA_SESSION=None) -> Generator[SQLA_SESSION, None, None]:
+def context(sess: SQLA_SESSION = None) -> Generator[SQLA_SESSION, None, None]:
```

Comment:
(Same comment as in srcflib.email on `SQLA_SESSION` capitalisation applies here)